modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,135 @@
+. In the YAML file for the VMI, add the label and a value for exposing the service for SSH connections. Enable the `masquerade` feature for the interface.
+
++
+.Example output
+[source,yaml]
+----
+...
+kind: VirtualMachineInstance
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-ssh <1>
+  name: vmi-ssh
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - masquerade: {} <2>
+        name: testmasquerade <3>
+        rng: {}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  networks:
+  - name: testmasquerade
+    pod: {}
+...
+----
+<1> This label is used by the service to identify the virtual machine instances that are enabled for SSH traffic connections. The label can be any `key:value` pair that is added as a `label` to this YAML file and as a `selector` in the service YAML file.
+<2> The interface type is `masquerade`.
+<3> The name of this interface is `testmasquerade`.
+
+. Create the VMI:
++
+[source,terminal]
+----
+oc create -f <path_for_the_VMI_YAML_file>
+----
++
+. In the YAML file for the service, specify the service name, port number, and the target port.
+
++
+.Example output
+[source,yaml]
+----
+...
+apiVersion: v1
+kind: Service
+metadata:
+  name: svc-ssh <1>
+spec:
+  ports:
+    - port: 20022 <2>
+      targetPort: 22 <3>
+      protocol: TCP
+  selector:
+    special: vmi-ssh <4>
+  type: NodePort
+...
+----
+<1> Name of the SSH service.
+<2> The port number that is assigned to direct SSH connections to the VMI.
+<3> The target port number for the service must match the port number for the VMI's interface.
+<4> The label name must match the label specified in the YAML file for the VMI.
+
+. Create the YAML file for the service.
++
+[source,terminal]
+----
+oc create -f <path_for_the_service_YAML_file>
+----
++
+. Verify that the VMI is running:
++
+[source,terminal]
+----
+oc get vmi
+----
++
+.Example output
+[source,terminal]
+----
+NAME    AGE     PHASE       IP              NODENAME
+vmi-ssh 6s      Running     10.244.196.152  node01
+----
+
+. Check the service to find out which port was acquired by the service:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+[source,terminal]
+----
+NAME            TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)           AGE
+svc-ssh-exp     NodePort   10.106.236.208 <none>        20022:30093/TCP   22s
+----
++
+In this example, the service acquired the port number 30093 and the IP address for the node is 10.106.236.208.
+
+. Log in to the VMI via SSH. Use the `ipaddress` of the node and the port number that display when the `oc get svc` command is used. The following example shows the `ssh` command with the node's IP address and the port number.
++
+[source,terminal]
+----
+$ ssh username@10.106.236.208 -p  30093
```

Comment:
As I noted [here](https://github.com/openshift/openshift-docs/pull/31224/files#r615359292), this is not the IP of the node, and another IP should be used. Since this is just an example I think a random IP can be used like: `192.168.66.101`.